src/class-wpml-cornerstone-register-strings.php

```diff
@@ -19,4 +19,12 @@ protected function register_strings_for_modules( array $data_array, array $packa
 	private function get_node_id( $data ) {
 		return md5( serialize( $data ) );
 	}
+
+	private function type_is_layout( $type ) {
+		return in_array(
```

Comment:
@dgwatkins please pass the "strict" argument as `true`.

Reply:
done :+1: